lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1643,6 +1650,13 @@ def _resource_sku_to_capability(  # noqa: C901
 
         # fill supported features
         azure_raw_capabilities: Dict[str, str] = {}
+        for location_info in resource_sku.location_info:
```

Comment:
What is this for? Is it related? if not, move it to a separated PR.

Reply:
The SKU capability for Ultra Disk is stored under location info because sometimes it is only available in certain availability zones.

``` yaml
"locationInfo": [
{
  "location": "WestUS3",
  "zoneDetails": [
    {
      "Name": [
        "1",
        "3",
        "2"
      ],
      "capabilities": [
        {
          "name": "UltraSSDAvailable",
          "value": "True"
        }
      ],
      "name": null
    }
  ],
  "zones": [
    "1",
    "3",
    "2"
    ]
  }
],
```

Reply:
I may wait to add this until availability zones are supported by LISA. If we set data disk to Ultra Disk and leave location blank, it would be better to select location where availability zones are not necessary / supported.